tests/pipelines.py

```diff
@@ -1,8 +1,19 @@
 """
 Some pipelines used for testing
 """
+from scrapy import ItemPipeline
 
-class ZeroDivisionErrorPipeline(object):
+
+class LegacyZeroDivisionErrorPipeline(object):
```

Comment:
isn't it better to keep the same old test name and add a deprecated warning or something? Then give the new test a name like BaseclassZeroDivisionErrorPipeline